src/prepack-cli.js

```diff
@@ -228,6 +230,10 @@ function run(
           reproFilePath = args.shift();
           // skip for repro purposes
           break;
+        case "cpuprofile":
+          cpuprofilePath = args.shift();
+          // skip for repro purposes
```

Comment:
This leaves me perplexed rather than enlightened.

Reply:
What exactly is perplexing?

Reply:
The comment. I now notice it is following an existing pattern. Nevertheless, what on earth is being skipped  and how does skipping help with repro purposes?

Reply:
Coming back to me now. Perhaps "do not include this in options produced by --repro".

Reply:
"Skipping" it means the repro-command-line that is synthesized is not going to mention that cpuprofile path. My reasoning for not including is that 1) it's really unlikely to affect behavior, and 2) this output path might not be portable across environments.

Reply:
Will make comments more enlightening...